package.json

```diff
@@ -632,12 +632,12 @@
     "fs-extra": "^7.0.1",
     "git-fetch-pack": "^0.1.1",
     "git-transport-protocol": "^0.1.0",
-    "hasha": "^3.0.0",
-    "js-yaml": "^3.12.1",
-    "opn": "^5.4.0",
+    "hasha": "^5.0.0",
+    "js-yaml": "^3.13.1",
+    "opn": "^6.0.0",
```

Comment:
This package is deprecated. We will switch to `open` package for the next release. Just adding for reference.